timbermill-server/src/main/java/com/datorama/timbermill/server/service/TimbermillController.java

```diff
@@ -20,47 +18,26 @@
 import org.springframework.web.bind.annotation.*;
 import org.springframework.web.util.ContentCachingRequestWrapper;
 
-import com.datorama.oss.timbermill.unit.Event;
-import com.datorama.oss.timbermill.unit.EventsWrapper;
-import com.google.common.cache.Cache;
-import com.google.common.cache.CacheBuilder;
+import com.datorama.oss.timbermill.unit.EventsList;
 
 @RestController
 public class TimbermillController {
 
 	private static final Logger LOG = LoggerFactory.getLogger(TimbermillController.class);
 
 	private static ExecutorService executorService;
-	private final Cache<String, String> idsCache;
 
 	@Autowired
 	private TimbermillService timbermillService;
 
 	public TimbermillController(@Value("${RECEIVING_THREAD:10}") int numOfThreads) {
-		CacheBuilder<String, String> cacheBuilder = CacheBuilder.newBuilder().weigher((key, value) -> key.length() + value.length());
-		idsCache = cacheBuilder
-				.maximumWeight(1000000) //1MB
-				.expireAfterWrite(10, TimeUnit.SECONDS)
-				.build();
 		executorService = Executors.newFixedThreadPool(numOfThreads);
 	}
 
+
 	@RequestMapping(method = RequestMethod.POST, value = "/events")
```

Comment:
@oz-zafar Let's add a new controller and make sure the new client sends to it
we need to keep the old one so that services that still have the old client will not fail